requirements.txt

```
@@ -1,2 +1,3 @@
 sopel>=7.1,<8
-google-api-python-client>=1.5.5,<1.8
+google-api-python-client>=1.5.5,<1.8;python_version<3.6
+google-api-python-client>=1.5.5,<3;python_version>=3.6
```

`pip` can't parse these; the format is invalid. Let's use the same style as the main Sopel repo.

```suggestion
google-api-python-client>=1.5.5,<1.8; python_version < '3.6'
google-api-python-client>=1.5.5,<3; python_version >= '3.6'
```